server/modules/survey/manager/surveyManager.js

```diff
@@ -149,10 +146,7 @@ export const importSurvey = async (params, client = db) => {
     // Add user to survey admins group (if not system admin)
     if (!User.isSystemAdmin(user)) {
       await UserManager.addUserToGroup(
-        user,
-        surveyId,
-        AuthGroup.getUuid(Survey.getAuthGroupAdmin(surveyInfo)),
-        User.getUuid(user),
+        { user, surveyId, groupUuid: AuthGroup.getUuid(Survey.getAuthGroupAdmin(surveyInfo)), userToAdd: user },
```

Comment:
maybe this can be extracted to some function as it is repeated on the line 111

Reply:
let me check it, I haven't noticed it...

Reply:
done ;)